src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -129,8 +129,7 @@ def validate_client_parameters(cmd, namespace):
             if is_storagev2(prefix):
                 from azure.cli.core._profile import Profile
                 profile = Profile(cli_ctx=cmd.cli_ctx)
-                n.token_credential, _, _ = profile.get_login_credentials(
-                    resource="https://storage.azure.com", subscription_id=n._subscription)
+                n.token_credential, _, _ = profile.get_login_credentials(subscription_id=n._subscription)
```

Comment:
`scope` is now managed by [Track 2 Storage SDK](https://github.com/Azure/azure-sdk-for-python/blob/900aa7d23212f528aec352b9bff2a87cd5294720/sdk/storage/azure-storage-blob/azure/storage/blob/_shared/constants.py#L25) (azure/multiapi/storagev2/blob/v2019_12_12/_shared/constants.py:25):

```py
STORAGE_OAUTH_SCOPE = "https://storage.azure.com/.default"
```